src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.util.DbConnector;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * @author Sergey Klunniy
+ */
+@Dao
+public class ItemDaoJdbcImpl implements ItemDao {
+
+    public static void main(String[] args) {
+        ItemDaoJdbcImpl itemDaoJdbc = new ItemDaoJdbcImpl();
+        Item item = new Item(15L, "яблоко", 55., "США");
+        itemDaoJdbc.update(item);
+    }
+
+    @Override
+    public Item update(Item item) {
+        try (Connection connection = DbConnector.connect()) {
+
+            String sql = String.format("UPDATE shop.items SET name='%s', price=" + item.getPrice() + ", description='%s' WHERE item_id=%d",
+                    item.getName(), item.getDescription(), item.getId());
+            Statement statement = connection.createStatement();
+            statement.executeUpdate(sql);
+
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return item;
+    }
+
+    @Override
+    public boolean delete(Item item) {
+        try (Connection connection = DbConnector.connect()) {
+            Statement statement = connection.createStatement();
+            String name = item.getName();
+            String sql = "DELETE FROM shop.items WHERE item_id =" + item.getId();
+            statement.execute(sql);
```

Comment:
```suggestion
            statement.executeUpdate(sql);
```
executeUpdate варто використовувати у випадках, коли твій запит до БД змінить її дані (INSERT, UPDATE, DELETE, DROP, ...)

Reply:
я якраз update роблю в цьому методі